.github/workflows/clang-tidy.yml

```
@@ -56,7 +55,7 @@ jobs:
         run: |
           echo -e "Checking Clang-Tidy $(clang-tidy --version)\n"
           touch source-check.log
-          for file in $(git diff --name-only ${{ github.event.pull_request.base.sha }} | grep -E '\.cpp$'); 
+          for file in $(git diff --name-only ${{ github.event.pull_request.head.sha }} ${{ github.event.pull_request.base.sha }} | grep -E '\.cpp$'); 
```

Can you try
```suggestion
          for file in $(git diff --name-only --merge-base ${{ github.event.pull_request.base.ref }} ${{ github.event.pull_request.head.sha }} | grep -E '\.cpp$'); 
```
for those of us using forks & don't always have an up-to-date base branch?